install_ruby.sh

```diff
@@ -76,6 +76,7 @@ fi
     --prefix=/usr/local \
     --disable-install-doc \
     --enable-shared \
+    --enable-yjit \
```

Comment:
Doesn't this fail when building older rubies, e.g 2.7.5 or 3.0.3?

Reply:
I tested this branch with 2.7.6 and it built fine. It looks like Ruby ignores flags on compile it doesn't know about. I confirmed that passing `--enable-yjit` to `ruby --enable-yjit -v` does throw an error. I also tested a random flag unknown to Ruby to confirm that it ignores unknown flags on compile, so this should be fine to add to all Rubies.

Reply:
👌